titiler/api/endpoints/cog.py

```diff
@@ -60,7 +60,7 @@ async def cog_bounds(
     response_model_exclude_none=True,
     responses={200: {"description": "Return basic info on COG."}},
 )
-async def cog_info(
+def cog_info(
```

Comment:
Switched to `def` so this can be reused in `/mosaicjson/info`.  These should all be normal `def` anyway since they aren't using coroutines.